outbox-kafka-spring/src/main/java/one/tomorrow/transactionaloutbox/repository/OutboxLockRepository.java

```diff
@@ -89,7 +89,7 @@ private boolean acquireOrRefreshLock(String ownerId, Duration timeout, Transacti
             }
 
             txStatus.flush();
-            logger.info("Acquired or refreshed outbox lock for owner {}, valid until {}", ownerId, validUntil);
```

Comment:
As far as I understand, this is used to monitor if the outbox publishing is still working. But I agree with you, it's logged very often. Maybe we can add a metric which can be used as alerting on application side if needed? WDYT @magro @danielrehmann

Reply:
Kindly reminder: @magro @danielrehmann

Reply:
The only thing that I'd find useful is to additionally change the `logger.debug("No outbox lock found. Creating one for {}", ownerId);` to `log.info` then, so that it's visible, which instance holds the lock.

Reply:
An additional metric could also be useful, I'd say this could be added in a separate PR.

Reply:
> The only thing that I'd find useful is to additionally change the `logger.debug("No outbox lock found. Creating one for {}", ownerId);` to `log.info` then, so that it's visible, which instance holds the lock.

Done :-)